tools/cli/src/logger.ts

```diff
@@ -0,0 +1,112 @@
+import { LogLevel, Logger } from '@backtrace/sourcemap-tools';
+import { format } from 'util';
+
+export interface LoggerOptions {
+    readonly verbose?: boolean[];
+    readonly quiet?: boolean;
+    readonly 'log-level'?: CliLogLevel;
```

Comment:
Out of curiosity, why not just use camelCase?

Reply:
I wanted to do args in POSIX commands style, and I've never seen a command with camelCase switches.

For example, exempt from `man rsync`:
```
-L, --copy-links            transform symlink into referent file/dir
--copy-unsafe-links     only "unsafe" symlinks are transformed
--safe-links            ignore symlinks that point outside the tree
-k, --copy-dirlinks         transform symlink to dir into referent dir
-K, --keep-dirlinks         treat symlinked dir on receiver as dir
```